### DISCUSSION TITLE
How/when are releases in the repo reflected in npm?

### DISCUSSION THREAD

**Discussion by kylevillegas93** [aws/aws-cdk#17886] [Q&A] [answered]
Hi!

Once changes are merged and deployed in the aws-cdk repo on Github, when and how are these changes reflected in the npm repository package? Will changes be picked up only in the most recent version of aws-cdk?

Thanks!
Kyle

**Comment by skinny85** [comment, marked as answer]
Hey @kylevillegas93,

the CDK packages for all languages, including JavaScript/TypeScript hosted on NPM, are released roughly once a week (you can check out our previous releases here: https://github.com/aws/aws-cdk/releases).

Yes, the changes merged since the last release will only be visible in the latest version of the released packages.

Thanks,
Adam

**Reply by kylevillegas93** [reply]
Thanks @skinny85 !!
